examples/template/README.rst

```diff
@@ -0,0 +1,12 @@
+
+==================================================================
+                        tmt test template
+==================================================================
+
+When creating a new ``tmt`` test it is recommended to use the test
+template provided in this directory. In order to be able to use it
+directly from the ``tmt test create`` command you might want to
+symlink it to your configuration::
+
+    ln -sr test.sh ~/.config/tmt/templates/script/tmt.j2
+    ln -sr main.fmf ~/.config/tmt/templates/test/tmt.j2
```

Comment:
This is using the user-configuration (thought it was still a [draft](https://tmt.readthedocs.io/en/stable/stories/cli.html#user)), but for these templates it would make more sense to be on the `project` level since each project might have slightly different `beakerlib` scripts. That doesn't seem to be in the spec/stories, and I'm not sure where a project's template files would reside. The only place I can think of would be `path/to/fmf_root/.fmf/templates/`